example/lib/main_screen.dart

```diff
@@ -24,7 +24,7 @@ class _MainScreenState extends State<MainScreen> {
         // ignore: lines_longer_than_80_chars
         'Hey this is a QR code. Change this value in the main_screen.dart file.';
 
-    final qrFutureBuilder = FutureBuilder<ui.Image>(
+    /*final qrFutureBuilder = FutureBuilder<ui.Image>(
```

Comment:
Remove commented code?

Reply:
Left for an example on preloading an image